src/Reader/Entry/Rss.php

```diff
@@ -84,7 +89,9 @@ public function getAuthors()
             return $this->data['authors'];
         }
 
-        $authors   = [];
+        $authors = [];
+
+        /** @psalm-suppress PossiblyNullReference */
```

Comment:
Similar to other `getExtension()` usages, these suppressions should IMO be removed and/or isolated to a new `private function getDublinCoreExtension()`